handlers/challenge_handler.py

```diff
@@ -473,6 +540,14 @@ def init(self, slack_wrapper):
 
             if not channel['is_archived'] and purpose and "ota_bot" in purpose and purpose["type"] == "CTF":
                 ctf = CTF(channel['id'], purpose['name'])
+
+                if "cred_user" in purpose:
```

Comment:
you could get rid of the conditionals and do:

`ctf.cred_user = purpose.get("cred_user", "")` if you're OK with the attribute always being set to _something_

Reply:
Didn't know that, but I'd also prefer setting it to an empty string instead of null/none.

Reply:
This does explicitly that. The second argument to `get` is what it should return if the key is missing.